app/notifications/user_sing_up_notification.rb

```diff
@@ -0,0 +1,35 @@
+class UserSingUpNotification < Noticed::Base
+
+  deliver_by :database, debug: true
+
+
+  deliver_by :database, if: :database_notifications?
+  deliver_by :email, mailer: "UserMailer", method: "created_user", if: :email_notifications?
+
+  # Variables that tells the system if it should send a notification if the recipient isn't subscribed
+  DEFAULT_SEND_NOTIFICATION = true
+  DEFAULT_SEND_EMAIL_NOTIFICATION = true
+
+  # Add required params
+  param :event_user
+
+  def message
+    "A new user #{params[:event_user]} with email #{params[:event_user].email} has been Sign Up."
```

Comment:
```suggestion
    "A new user #{params[:event_user]} with email #{params[:event_user].email} has signed up."
```
Minor grammar correction